libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -83,7 +104,43 @@ export const TicketReplyForm: React.FC<{ ticketId: string }> = ({
               maxSize={3145728}
             />
 
+            <div className="status-wrapper">
+              <Label>Ticket Status</Label>
+              {ticketData?.Status !== 'resolved' ? (
+                <div className="status-checkbox">
+                  <Field type="checkbox" name="status" />
+                  My issue has been resolved
+                </div>
+              ) : (
+                <div className="status-checkbox">
+                  <Field type="checkbox" name="status" checked disabled /> My
+                  issue has been resolved
+                </div>
+              )}
```

Comment:
**Redundant Code.** This can cause incomplete updates e.g. updating identical message text in one place but not in another.

<details><summary>Suggestion 1</summary>

```suggestion
              <div className="status-checkbox">
                <Field
                  type="checkbox"
                  name="status"
                  checked={ticketData?.Status === 'resolved'}
                  disabled={ticketData?.Status === 'resolved'}
                />{' '}
                My issue has been resolved
              </div>
```

</details>
<details><summary>Suggestion 2</summary>

Store `{ticketData?.Status === 'resolved'}` in a variable, so the logic is in only one place. If it is different than `isResolved`, distinguish the two variables clearly.

</details>

Reply:
I tried this in cd5aed8 (initial commit of #404, removed via b2f9cf5). It didn't seem to work for me — with ticket open, checking checkbox had no effect. Maybe I did it wrong. Maybe there is a reason for the code as is.

Reply:
I'm willing to forgo a solution. Feel free to resolve.

Reply:
React and Checkboxes is a quagmire last time I investigated. And this case adds Formik to the mix. Don't worry about it. I resolve this thread.

Instead, we can have DRY-er code via https://github.com/TACC/tup-ui/pull/404#discussion_r1448115997.